docs/index.rst

```diff
@@ -1,8 +1,3 @@
-..  -*- coding: utf-8 -*-
-
-.. include:: ../README.md
-    :parser: myst_parser.sphinx_
```

Comment:
this should not have been deleted

Reply:
Which part in particular, just those first two lines?

Reply:
Ok done!